addons/config/configs/CfgAmmo.hpp

```diff
@@ -0,0 +1,262 @@
+        submunitionConeAngle = 22; // 10
+        submunitionConeType[] = {"random",70}; // "randomcenter",50
+    };
+
+    class Mo_cluster_Bomb_01_F;
+    class Mo_cluster_Bomb_03_F : Mo_cluster_Bomb_01_F // BL-778 HE Cluster Submunition
+    {
+        aiAmmoUsageFlags = "64 + 128"; // "64 + 128 + 256"
+        airFriction = -0.00005; // -0.0005
+        caliber = 0.5; // 34
+        hit = 25; // 70
+        indirectHit = 19; // 12
+        indirectHitRange = 10; // 6
+        suppressionRadiusHit = 65; // 30
+    };
+
+    class BombCore;
+    class Bo_Mk82 : BombCore // Mk-82 Airburst
+    {
+        cameraViewAvailable = 1; // 0
+        indirectHit = 70; // 1100
+        indirectHitRange = 75; // 12
+        suppressionRadiusHit = 210; // 100
+    };
+
+    class ammo_Bomb_SmallDiameterBase;
+    class ammo_Bomb_SDB : ammo_Bomb_SmallDiameterBase // SDB Bomb
+    {
+        cameraViewAvailable = 1; // 0
+    };
+
+    class ammo_Penetrator_Base;
+    class ammo_Penetrator_AGM_02 : ammo_Penetrator_Base // AGM-65 L Penetrator
+    {
+        caliber = 145; // 66.7
+        indirectHit = 350; // 85
+        indirectHitRange = 2; // 8
+    };
+
+    class ammo_Penetrator_Scalpel : ammo_Penetrator_Base // AGM-114K Penetrator
+    {
+        cameraViewAvailable = 1; // 0
+    };
+
+    class M_Scalpel_AT;
+    class ACE_Hellfire_AGM114K : M_Scalpel_AT // AGM-114K
+    {
+        cameraViewAvailable = 1; // 0
+        indirectHitRange = 1; // 4
+        trackOversteer = 0.95; // 1
+
+        class CamShakeExplode {
+            distance = 300;
+            duration = 1.5;
+            frequency = 18;
+            power = 16;
+        };
+        class CamShakeFire {
+            distance = 150;
+            duration = 1.5;
+            frequency = 18;
+            power = 4;
+        };
+        class CamShakeHit {
+            distance = 1;
+            duration = 0.6;
+            frequency = 20;
+            power = 60;
+        };
+        class CamShakePlayerFire {
+            distance = 1;
+            duration = 0.1;
+            frequency = 20;
+            power = 2;
+        };
+    };
+
+    class BulletBase;
+    class Gatling_30mm_HE_Plane_CAS_01_F : BulletBase // A-10 GAU-8
+    {
+        caliber = 19; // 1.4
+        indirectHit = 35; // 12
+        indirectHitRange = 4.5; // 3
+    };
+
+    class B_30mm_HE;
+    class B_40mm_GPR : B_30mm_HE // 40mm GPR
+    {
+        hit = 40; // 70
+    };
+
+    class B_30mm_APFSDS;
+    class B_40mm_APFSDS : B_30mm_APFSDS // 40mm APFSDS
+    {
+        hit = 100; // 150
+    };
+
+    class DirectionalBombBase;
+    class ClaymoreDirectionalMine_Remote_Ammo : DirectionalBombBase // Claymore
+    {
+        indirectHit = 50; // 40
+        indirectHitRange = 80; // 30
+        /*mineTrigger = {"RemoteTrigger","TimeTrigger","ACE_MagneticTrigger","IRTrigger"};*/
+        mineTrigger = "IRTrigger";
+    };
```

Comment:
Da wollten wir nochmal schauen

Reply:
Hab ich bereits in einem Commit, den ich noch hochlade, rausgeschmissen.